src/pages/task/logs/logMessageLine/LogLines.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from "@emotion/styled/macro";
+
+export const DebugMessage = styled.div`
+  color: #666;
+`;
+
+export const InfoMessage = styled.div`
+  color: #333;
+`;
+
+export const WarnMessage = styled.div`
+  color: #ffa500;
+`;
+
+export const ErrorMessage = styled.div`
+  color: #ff0000;
+`;
```

Comment:
instead of having 4 styled divs, this could be one component that accepts a `color` prop